implementations/micrometer-registry-influx/src/main/java/io/micrometer/influx/CreateBucketJSONBuilder.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2020 VMware, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.influx;
+
+import io.micrometer.core.lang.Nullable;
+
+/**
+ * Builds a JSON Bucket representation for InfluxDB 2.
+ *
+ * @author Jakub Bednar
+ */
+class CreateBucketJSONBuilder {
+
+    private static final String BUCKET_TEMPLATE = "{\"name\":\"%s\",\"orgID\":\"%s\",\"retentionRules\":[%s]}";
```

Comment:
It seems we need to generate a valid orgID, which is different from the org name. Passing an org name here results in the following error being returned:

```
{
	"code": "invalid",
	"message": "failed to unmarshal json: id must have a length of 16 bytes"
}
```

Reply:
We have to check if the `org` parameter is `ID` otherwise retrieve `ID` by something like: [retrieveOrg](https://github.com/micrometer-metrics/micrometer/blob/229dcc3f4f2ab79e04fc85e1c22379323ee589cf/implementations/micrometer-registry-influx/src/main/java/io/micrometer/influx/InfluxMeterRegistry.java#L338)
or It is a little bit out of scope of and we can remove the support for creating Bucket.

Reply:
> It is a little bit out of scope of and we can remove the support for creating Bucket.

I'm leaning towards this direction. Perhaps we can revisit that option later. For initial v2 support, I think it is fine to not have it. Most companies will want to configure a retention policy on the bucket when creating it, and they will make tokens with specific access, so it seems highly likely that the token configured in the app won't have permission to create buckets anyways.

Reply:
Ok, I will remove support for creating Bucket.

Reply:
The support for creating `Bucket` is removed.